packages/services/src/services/CloudAccountAuth/__tests__/CloudAccountAuth.test.js

```diff
@@ -1,30 +1,41 @@
 /**
- * Copyright IBM Corp. 2021
+ * Copyright IBM Corp. 2021, 2022
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 import CloudAccountAuthAPI from '../CloudAccountAuth';
+import root from 'window-or-global';
 
-describe('CloudAccountAuth cookie utility', () => {
-  it('should fetch the CloudAccountAuth cookie and return the authenticated string', () => {
-    Object.defineProperty(window.document, 'cookie', {
-      writable: true,
-      value: 'com.ibm.cloud.iam.LoggedIn.prod=1',
-    });
+describe('CloudAccountAuth personalization utility', () => {
+  it('should fetch the personalization window boolean and return the authenticated string', () => {
+    root.digitalData = {
+      ddo: {
```

Comment:
no need for the `ddo` here! we're reading from `digitalData.user.segment.isCloudLoggedOn`